include/bluetooth/hci.h

```diff
@@ -96,8 +97,8 @@ struct bt_hci_iso_ts_data_hdr {
 #define BT_HCI_ISO_TS_DATA_HDR_SIZE     8
 
 struct bt_hci_iso_hdr {
-	uint16_t handle;
-	uint16_t len;
+	uint16_t handle; /* 12 bit handle, 2 bit PB flags, 1 bit TS_Flag */
```

Comment:
Should there be a comment about the RFU bit?

Reply:
Added